php_phongo.c

```diff
@@ -1390,8 +1390,6 @@ static mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri, zv
 	const char                *mongoc_version, *bson_version;
 	mongoc_client_t           *client;
 
-	ENTRY;
```

Comment:
I assume `php_phongo_make_mongo_client()` was the only use of libmongoc tracing in PHPC. Is that correct?